streams/src/test/java/org/apache/kafka/test/ProcessorTopologyTestDriver.java

```diff
@@ -187,7 +185,7 @@ public ProcessorTopologyTestDriver(StreamsConfig config, TopologyBuilder builder
 
         // Set up all of the topic+partition information and subscribe the consumer to each ...
         for (String topic : topology.sourceTopics()) {
-            TopicPartition tp = new TopicPartition(topic, 1);
+            TopicPartition tp = new TopicPartition(topic, 0);
```

Comment:
nit: I'll be better to declare `partitionId` as this class's static field, along with `taskId` to be also static field which are always initialized as `0` and `new TaskId(0, partitionId)` so that future extension wont mess them up again.